templates/docs/examples/templates/maas-docs-grid.html

```diff
@@ -52,113 +31,119 @@
-            <a class="p-side-navigation__link" href="#">Power management</a>
-          </li>
-          <li class="p-side-navigation__item ">
-            <a class="p-side-navigation__link" href="#">Commision machines</a>
-          </li>
-          <li class="p-side-navigation__item">
-            <span class="p-side-navigation__text is-selected">Testing</span>
-            <ul class="p-side-navigation__list">
-              <li class="p-side-navigation__item">
-                <a class="p-side-navigation__link" href="#">Hardware testing</a>
-              </li>
-              <li class="p-side-navigation__item ">
-                <a class="p-side-navigation__link is-active" href="#">Network testing</a>
-              </li>
-              <li class="p-side-navigation__item ">
-                <a class="p-side-navigation__link" href="#">Commissioning and hardware testing scripts</a>
-              </li>
-            </ul>
-          </li>
-          <li class="p-side-navigation__item ">
-            <a class="p-side-navigation__link" href="#">Deploy machines</a>
-          </li>
-          <li class="p-side-navigation__item ">
-            <a class="p-side-navigation__link" href="#">Tags</a>
-          </li>
-        </ul>
 
-        <ul class="p-side-navigation__list">
-          <li class="p-side-navigation__item--title">
-            <a class="p-side-navigation__link">Troubleshoot</a>
-          </li>
-          <li class="p-side-navigation__item">
-            <a class="p-side-navigation__link" href="#">Getting help</a>
-          </li>
-          <li class="p-side-navigation__item ">
-            <a class="p-side-navigation__link" href="#">Non-snap MAAS installs</a>
-          </li>
-          <li class="p-side-navigation__item ">
-            <a class="p-side-navigation__link" href="#">MAAS 2.5 (and earlier) documentation</a>
-          </li>
-          <li class="p-side-navigation__item">
-            <a href="#" class="p-side-navigation__link is-selected">Upgrading MAAS</a>
-            <ul class="p-side-navigation__list">
-              <li class="p-side-navigation__item">
-                <a class="p-side-navigation__link" href="#">Upgrade from 2.3 to 2.4</a>
-              </li>
-              <li class="p-side-navigation__item ">
-                <a class="p-side-navigation__link" href="#">Upgrade from 1.9 to 2.x</a>
-              </li>
-            </ul>
-          </li>
-        </ul>
+        <button class="p-side-navigation__toggle js-drawer-toggle" aria-controls="drawer">
+          <span>Toggle side navigation</span>
+        </button>
+
+        <div class="p-side-navigation__overlay js-drawer-toggle" aria-controls="drawer"></div>
+
+        <div class="p-side-navigation__drawer" id="drawer">
+          <div class="p-side-navigation__drawer-header">
+            <button class="p-side-navigation__toggle--in-drawer js-drawer-toggle" aria-controls="drawer">
+              <span>Toggle side navigation</span>
```

Comment:
Is this span required?

Reply:
`p-button has-icon` requires span, and this is based on it.
But because we always have icon on the left I think `<span>` may not be needed. I'll have a look. 

Good catch, thanks.